stores/common_test.go

```diff
@@ -686,6 +686,9 @@ func testBasicSubStore(t *testing.T, s Store) {
 }
 
 func testGetSeqFromStartTime(t *testing.T, s Store) {
+	limits := testDefaultStoreLimits
+	limits.MaxAge = 500 * time.Millisecond
```

Comment:
If you look down, you will see that we are sending 100 messages, each one with a 1ms delay. So no, 10ms would not work. Also, be careful to be too tight because then you start to get Travis flappers ;-)